tools/genomespace/genomespace_exporter.xml

```diff
@@ -1,54 +1,42 @@
 <?xml version="1.0"?>
-<tool name="GenomeSpace Exporter" id="genomespace_exporter" require_login="True" version="0.0.4">
+<tool name="GenomeSpace Exporter" id="genomespace_exporter" version="0.0.4">
     <description> - send data to GenomeSpace</description>
-    <command interpreter="python">genomespace_exporter.py 
-        --genomespace_site "prod"
-        #assert $__user__, Exception( 'You must be logged in to use this tool.' )
-        #set $username = $__user__.preferences.get( 'genomespace_username', None )
-        #set $token = $__user__.preferences.get( 'genomespace_token', None )
-        #assert None not in ( $username, $token ), Exception( 'You must associate a GenomeSpace OpenID with your account and log in with it.' )
+    <command interpreter="python">genomespace_exporter.py
+        #set $target_folder = $genomespace_browser.split("^")[0]
+        #set $token = $genomespace_browser.split("^")[1] or $__user__.preferences.get( 'genomespace_token', None )
+        
+        #assert $target_folder, Exception( 'You must select a valid target folder.' )
+        #assert $token, Exception( 'Invalid token. You must be logged into GenomeSpace through OpenID or select a valid folder via the GenomeSpace browse dialog.' )
         #import binascii
-        --username "${username}"
-        --token "${token}"
-        --dataset "${input1}"
-        #if $subdirectory:
-            #for $subd in str( $subdirectory ).split( '/' ):
-                #if not $subd:
-                    --subdirectory "${ binascii.hexlify( '/' ) }"
-                #else:
-                    --subdirectory "${ binascii.hexlify( $subd ) }"
-                #end if
-            #end for
-        #else:
-            --subdirectory "${ binascii.hexlify( 'galaxy_export' ) }"
-            --subdirectory "${ binascii.hexlify( str( $base_url ).split( '://', 1 )[-1] ) }" ##Protocol removed by request
-        #end if
+        --token "${ binascii.hexlify(str(token).encode('utf-8')) }"
+        --input_file "${ binascii.hexlify( str($input1).encode('utf8') ) }"
```

Comment:
Same here, everything that does not need to be expanded by the shell should be in single quotes